plugin-common/src/test/java/org/opennms/plugins/cloud/testserver/GrpcTestServerInterceptor.java

```diff
@@ -59,9 +65,13 @@ public <ReqT, RespT> ServerCall.Listener<ReqT> interceptCall(ServerCall<ReqT, Re
 
     private boolean isTokenNeeded(final String methodName) {
         // the following calls don't need a token, all others do:
-        return ! ("org.opennms.dataplatform.access.Authenticate/AuthenticateKey".equals(methodName)
+        return !("org.opennms.dataplatform.access.Authenticate/AuthenticateKey".equals(methodName)
                 || "org.opennms.dataplatform.access.Authenticate/GetServices".equals(methodName)
                 || "org.opennms.dataplatform.access.Authenticate/GetAccessToken".equals(methodName)
                 || "org.opennms.dataplatform.access.Authenticate/RenewCertificate".equals(methodName));
     }
+
+    private boolean isTraceParentHeaderNeeded(final String methodName) {
+        return StringUtils.equals(methodName, "org.opennms.tsaas.Timeseries/Store");
```

Comment:
I am not sure but I assume we want to have the trace for all GRPC calls, no?
I would assume we make that a general feature: Add the trace whenever we make a GRPC call?

Reply:
Ok lets assume that